config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -187,3 +187,51 @@ periodics:
           limits:
             cpu: "2"
             memory: "6Gi"
+
+  # This jobs runs e2e.test with a focus on long running tests for all alpha/beta storage features on a kind cluster
+  - name: pull-kubernetes-e2e-storage-kind-alpha-beta-features-slow
```

Comment:
you copy pasted a presubmit job to the periodics field, you need to rename it and also update the config fields

it should be `ci-` or `periodic-`, `pull-kubernetes` implies it runs on prs to kubernetes/kubernetes, this type of job runs on a schedule

Reply:
I want to make it run in PRs but it should optional. Does the ci- or periodic- do that?